src/Backend/Request.php

```diff
@@ -25,21 +24,6 @@ public function __construct(ClientInterface $client)
     }
 
     /**
-     * @param string $url
-     * @param string $method
-     * @param array  $options
-     *
-     * @return RequestInterface
-     */
-    protected function createRequest($url, $method = 'GET', $options = [])
```

Comment:
I'll not discuss public/protected/private here. The internet is full of the pros and cons. While I agree with your definition on the theoretical basis, this is unfortunately not how it works in reality.

I'll create a bug fix and add the method back and hope for @compeak to make a new release then.